tensordict/prototype/tensorclass.py

```diff
@@ -359,7 +360,7 @@ def wrapper(self, key: str, value: Any) -> None:
                 f"Cannot set the attribute '{key}', expected attributes are {expected_keys}."
             )
 
-        if isinstance(value, _ACCEPTED_CLASSES):
+        if isinstance(value, tuple(tensordict.tensordict._ACCEPTED_CLASSES)):
```

Comment:
Perhaps we could maintain `_ACCEPTED_CLASSES` as a tuple rather than casting it every time we call `setattr`?

Reply:
I wanted it to be mutable to make it easy to have all modules see the changes no matter what we do with it.
Can we have the same result with a tuple?

Reply:
I believe so, because all modules are actually accessing the same variable, rather than different references to the same object. So reassigning that variable is seen by all.

I will validate that though.

Reply:
I think I may actually be wrong on this point. Let's leave it as it is. Casting to tuple for small number of elements like that is pretty cheap anyway.